src/plugins/data_table/data_table.js

```diff
@@ -58,6 +58,7 @@ var LocalDataTable = (function() {
       this._enableRowHighlight();
       this.paginate && this._initPaginationHandling();
       this._triggerChangeSelection();
+      this._colReorder && this._colReorder._fnSetupStickyTableHeader.call(this._colReorder, this._colReorder.s.iStickyTableHeader);
```

Comment:
shouldn't this happen in colReorder? any chance there is already an init where you can just call `_fnSetupStickyTableHeader ` ? 

If not.. all of this is private stuff in there.. what about:

```
this._colReorder && this._colReorder.stickyHeader.call(this._colReorder)
```

and inside that "public" method it does the stuff above? That's the 2nd fav option though.